quipucords/tests/api/aggregate_report/test_aggregate_report_model.py

```diff
@@ -124,14 +147,13 @@ def report_and_expected_aggregate() -> tuple[Report, AggregateReport]:  # noqa:
         infrastructure_type=SystemFingerprint.UNKNOWN,  # instances_unknown++
         deployment_report=deployments_report,
         cpu_count=None,  # cpu_count is only relevant for openshift sources
-        cpu_core_count=None,  # missing_cpu_core_count++
+        cpu_core_count=None,  # ansible has no effect on missing_cpu_core_count
```

Comment:
U probably know this but FYI: ansible & rhacs are not properly normalized. their data doesn't fit fingerprints.